slack-post.py

```diff
@@ -197,4 +197,7 @@ def main(args):
     args, extra = parser.parse_known_args()
     if args.command:
         args.command = [args.command] + extra
+    if not args.webhook_url:
+        sys.exit('Please either set the $SLACK_WEBHOOK_URL variable, or provide'
```

Comment:
Just discovered that the `ArgumentParser` class has a nice method called `error` ([doc here](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error)). Can you change `sys.exit` with `parser.error`? The advantage is that it prints also the **usage** of the command and we keep the output consistent with other errors of `argparse`.